data/sql/updates/pending_db_world/rev_1643215013369974869.sql

```diff
@@ -0,0 +1,50 @@
+(61255, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3385.7449, -12715.757, 17.479109, 2.68819427490234375, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3385.7449 -12715.757 17.479109
+(61256, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3351.1653, -12717.854, 17.147543, 4.127032279968261718, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3351.1653 -12717.854 17.147543
+(61257, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3384.7056, -12747.812, 12.962563, 2.626925945281982421, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3384.7056 -12747.812 12.962563
+(61258, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3320.3354, -12716.188, 24.402348, 3.298841476440429687, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3320.3354 -12716.188 24.402348
+(61259, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3383.158, -12686.646, 17.665024, 1.074158787727355957, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3383.158 -12686.646 17.665024
+(61260, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3318.929, -12750.321, 16.73596, 3.903232574462890625, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3318.929 -12750.321 16.73596
+(61261, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3320.553, -12687.106, 20.46702, 5.458890914916992187, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3320.553 -12687.106 20.46702
+(61262, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3381.1094, -12652.545, 28.912785, 4.199787139892578125, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3381.1094 -12652.545 28.912785
+(61263, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3385.4202, -12813.755, 13.007764, 2.13775491714477539, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3385.4202 -12813.755 13.007764
+(61264, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3411.1362, -12779.929, 14.255584, 5.14163970947265625, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3411.1362 -12779.929 14.255584
+(61265, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3418.1714, -12752.151, 10.181131, 4.816044807434082031, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3418.1714 -12752.151 10.181131
+(61266, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3352.9028, -12686.875, 20.251678, 6.248052120208740234, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3352.9028 -12686.875 20.251678
+(61267, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3348.0188, -12749.719, 14.26492, 0.948191642761230468, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3348.0188 -12749.719 14.26492
+(61268, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3417.4426, -12716.594, 16.267937, 2.189768791198730468, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3417.4426 -12716.594 16.267937
+(61269, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3412.6267, -12687.893, 22.139761, 5.748758792877197265, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3412.6267 -12687.893 22.139761
+(61270, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3381.6501, -12779.417, 14.582455, 5.741998672485351562, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0); -- .go xyz -3381.6501 -12779.417 14.582455
```

Comment:
only the last note is causing the codestyle as it doesn't check for comments... If you want it you can do a `/* */` comment inside the );

Reply:
oh we had this happen with a prior pr, its any not after the , or ; that is followed by a -- is what throws the error regardless if it is the last line or not in the code style test.

Reply:
yea that is why you can write it
```sql
(61270, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3381.6501, -12779.417, 14.582455, 5.741998672485351562, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0 /* .go xyz -3381.6501 -12779.417 14.582455-- .go xyz -3381.6501 -12779.417 14.582455*/);
```
For just the last line, as the CI always checks for `;` to be the last character on the last line of an SQL file

Reply:
that is good bit of info to know actually. hell i wish we knew about that bit about a month ago.

Reply:
I'd love for the CI to be able to ignore comments, but nobody knows how to do it yet